parkhangs-webapp/src/components/CommentForm.js

```diff
@@ -26,51 +22,55 @@ class CommentForm extends React.Component {
     handleAddComment = (comment) => {
         event.preventDefault()
 
-        const commentText =  this.state.comment || this.comment()
+        if (this.state.comments !== '') {
 
-        if ("parkId" in this.props) {
-          const parkId = this.props.parkId;
-          const newComment = {
-            parkId: parkId,
-            comment: commentText
-          }
-          this.props.addParkComment(newComment, this.props.user)
-        } else if ("eventId" in this.props) {
-          const eventId = this.props.eventId;
-          const newComment = {
-            eventId: eventId,
-            comment: commentText
+          const commentText =  this.state.comments
+
+          if ("parkId" in this.props) {
+            const parkId = this.props.parkId;
+            const newComment = {
+              parkId: parkId,
+              comment: commentText
+            }
+            this.props.addParkComment(newComment, this.props.user)
+          } else if ("eventId" in this.props) {
+            const eventId = this.props.eventId;
+            const newComment = {
+              eventId: eventId,
+              comment: commentText
+            }
+              this.props.addEventComment(newComment, this.props.user)
+            }
+            this.setState({
+              comments: ''
+            })
           }
-            this.props.addEventComment(newComment, this.props.user)
-        }
-        this.setState({
-            comments: ''
-          })
         }
 
     render() {
-        const comment = this.comment();
 
         return (
             <div className="CommentForm">
 
                 <form className="CommentForm">
                     <div className="formsection comment">
                         <label htmlFor="comment">Comment: </label>
-                        <textarea
-                            onChange={this.handleUpdateComment}
-                            id="comment"
-                            defaultValue={comment}/>
+                        <input
```

Comment:
is there a reason you converted it to an input rather than a textarea?

Reply:
the behavious was being weird when i used value instead of defaultValue for textarea, but switching to input fixed it for some reason

Reply:
yeah textarea has different inputs than an input I think...  do we want users to only be able to add comments in a small one line textbox?

Reply:
https://reactjs.org/docs/forms.html#the-textarea-tag

Reply:
I think we should keep text area if we can, since typically users want more than one line for input

Reply:
ahh I didn't realize a commment box wouldn't expand, good point claire